assets/base.css

```diff
@@ -1189,39 +1189,89 @@ deferred-media {
 .share-button {
   display: block;
   position: relative;
+  max-width: 31rem;
+}
+
+.share-button details {
+  width: fit-content;
+}
+
+.share-button__button {
+  font-size: 1.4rem;
+  display: flex;
+  align-items: baseline;
+  color: rgba(var(--color-link), var(--alpha-link));
+  margin-left: 0;
+  padding-left: 0;
+}
+
+.share-button__button:hover, details[open] > .share-button__button {
+  color: rgb(var(--color-link));
+}
+
+.share-button__button:hover {
+  text-decoration: underline;
+  text-underline-offset: 0.3rem;
+}
```

Comment:
I noticed this happening on desktop too: https://screenshot.click/10-36-0spw8-19pv7.mp4

Reply:
Chrome bug mentioned in this comment: https://github.com/Shopify/dawn/pull/349#discussion_r686297866